ddtrace/internal/hostname.py

```diff
@@ -1,11 +1,14 @@
 import functools
 import socket
+from typing import Any
+from typing import Callable
 
 
 _hostname = None
 
 
 def _cached(func):
```

Comment:
See https://github.com/DataDog/dd-trace-py/pull/2255 which removes this entirely

Reply:
Will update upstream 😃